### DISCUSSION TITLE
Performance differences between devices

### DISCUSSION THREAD

**Discussion by AJshere** [keiwando/toonsquid#143] [Q&A]
Hey! I'm planning on buying an iPad soon and I've been seeing some people complaining about performance here and there, could you say which device you as the reader have and tell your experience and what type of work you do? I'd thank you a lot if you do it.

**Comment by DinoAwakening** [comment]
My experience might not be useful since I have an M1 iPad Air. But yeah, still… Canvas above 4K used to be a problem (I think it rather was a problem on ToonSquid itself). Hopefully in 1.7.8 it won’t occur anymore xd
